MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+* otelcol_receiver_refused_metric_points
+* otelcol_scraper_errored_metric_points
+* otelcol_scraper_scraped_metric_points
+* scrape_duration_seconds
+* scrape_samples_post_metric_relabeling
+* scrape_samples_scraped
+* scrape_series_added
+* system.cpu.load_average.15m
+* system.cpu.load_average.1m
+* system.cpu.load_average.5m
+* system.cpu.time
+* system.disk.io
+* system.disk.io_time
+* system.disk.merged
+* system.disk.operation_time
+* system.disk.operations
+* system.disk.pending_operations
+* system.disk.weighted_io_time
+* system.filesystem.inodes.usage
+* system.filesystem.usage
+* system.memory.usage
+* system.network.connections
+* system.network.dropped
+* system.network.errors
+* system.network.io
+* system.network.packets
+* system.paging.faults
+* system.paging.operations
+* system.paging.usage
+* system.processes.count
+* system.processes.created
+* up
+
+### Changes in Objects in Splunk OpenTelemetry Connector for Kubernetes (SCK-OTEL)
+This chart currently does not support the collection of Kubernetes objects. This is future functionality. You can still use SCK to collect other objects with current functionality.
+
+## Migration Overview
+### Migration Options/Matrix
+ The following is a matrix for migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes:
+
+| Method  | Logs | Metrics | Objects |
+|---|---|---|---|
+| SCK | Yes | Yes | Yes |
+| Splunk OpenTelemetry Collector for Kubernetes | Yes | Yes | No |
+
+As shown above, you can acquire logs and metrics using Splunk OpenTelemetry Collector for Kubernetes. If you have objects currently deployed and need objects data, we recommend you leave your current SCK objects deployment as-is (helm command args --set="splunk-kubernetes-logging.enabled=false,splunk-kubernetes-metrics.enabled=false,splunk-kubernetes-objects.enabled=true").  In this way, you can use the SCK just for objects data together with Splunk OpenTelemetry Collector for Kubernetes for logs and other telemetry data. 
+### Checkpoint Translation
+WIth migration to the Splunk OpenTelemetry Collector for Kubernetes, the underlying framework/agent being used has changed, so there needs to be a baton handoff for checkpoint data so that the new OpenTelemetry agent can continue where Fluentd left off. All of this occurs automatically as an initContainer when you deploy the new helm chart in your cluster for the first time. As long as you properly configured the values.yaml file for the new helm chart, it will pick up where it left off. Without proper configuration for migration, there is a possibility of either data duplication or data loss while migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes. 
+ 
+ If you need to migrate Fluentd's position files again, you can delete the OpenTelemetry checkpoint files in the ```"/var/lib/otel_pos/"``` directory from Kubernetes nodes. Then restart the new helm chart Daemonet.
+ 
+## Step 1: Preparing Your values.yaml file for migration
```

Comment:
"Step 1: Preparing your ..."

"Translate the values.yaml file from SCK to an appropriate format for SCK-OTEL. The following are the configurations for SCK and SCK-OTEL:"

- SCK configuration (make this a hyperlink)
- Splunk OpenTelemetry Collector for Kubernetes configuration (make this a hyperlink)